src/ts/app/framework/src/css-scope.ts

```diff
@@ -1,4 +1,5 @@
 export enum CssScope {
+    CONSOLE = "CONSOLE",
```

Comment:
ok... could you put a link to how this stuff works? I am toatally missing it.

Reply:
We need to talk live - let's go through it next time. It's all to do w/ the shadow dom / css isolation. 

This [bottom section](https://lit-html.polymer-project.org/guide/styling-templates#rendering-in-shadow-dom) covers some of it.